tf-modules/archive/api.tf

```diff
@@ -27,9 +27,9 @@ locals {
       EARTHDATA_CLIENT_ID              = var.urs_client_id
       EARTHDATA_CLIENT_PASSWORD        = var.urs_client_password
       EcsCluster                       = var.ecs_cluster_name
-      EmsDistributionReport            = aws_lambda_function.ems_distribution_report.arn
-      EmsIngestReport                  = aws_lambda_function.ems_ingest_report.arn
-      EmsProductMetadataReport         = aws_lambda_function.ems_product_metadata_report.arn
+      EmsDistributionReport            = var.ems_deploy ? aws_lambda_function.ems_distribution_report[0].arn : null
```

Comment:
Why did all of these change to [0] ?

Reply:
Short answer: Terraform told me to do that!
Real answer: With adding the `count` parameter to EMS resources, Terraform needs to know which `ems_distribution_report` lambda we are referring to because there could be multiple. Since it's really only one we can just set it to the "first one."